doc/classes/CollisionObject2D.xml

```diff
@@ -68,6 +68,7 @@
 			<argument index="0" name="owner_id" type="int">
 			</argument>
 			<description>
+				Returns [code]true[/code] if one way collision is enabled for the shape owner of given index.
```

Comment:
The description for one way collision is really brief, so could I just copy it from CollisionPolygon2D? I'd also add it to line line 71.
Also, I'll fix the entire file to add [code] around the arguments in all methods descriptions. There are quite a few descriptions without those.

Reply:
That sounds good to me.